add channels last support for FractionalMaxPool2d and FractionalMaxPool3d

mingfeima opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61338 add channels last support for FractionalMaxPool2d and FractionalMaxPool3d**
* #62170 add channels last support for ReflectionPad2d and ReflectionPad3d
* #62169 add channels last support for ReplicationPad2d and ReplicationPad3d
* #60935 add channels last support for AdaptiveMaxPool3d on CPU
* #60620 add channels last support for BatchNorm3d on CPU
* #60619 add channels last support for AvgPool3d on CPU
* #60618 add channels last support for MaxPool3d on CPU
* #60617 add channels last support for AdaptiveAvgPool3d on CPU

Differential Revision: [D29663670](https://our.internmc.facebook.com/intern/diff/D29663670)

mingfeima committed:
add channels last support for FractionalMaxPool2d on CPU

[ghstack-poisoned]

mingfeima committed:
Update on "add channels last support for FractionalMaxPool2d on CPU"




[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29663670).

mingfeima committed:
Update on "add channels last support for FractionalMaxPool2d on CPU"


Differential Revision: [D29663670](https://our.internmc.facebook.com/intern/diff/D29663670)

[ghstack-poisoned]

mingfeima committed:
Update on "add channels last support for FractionalMaxPool2d and FractionalMaxPool3d"


Differential Revision: [D29663670](https://our.internmc.facebook.com/intern/diff/D29663670)

[ghstack-poisoned]
